lib/jekyll.rb

```diff
@@ -78,7 +78,7 @@ class << self
     # images and allows you to skip that when working in development.
 
     def env
-      ENV["JEKYLL_ENV"] || "development"
+      ENV["JEKYLL_ENV"] || "local"
```

Comment:
I thought about that, too. I think many of our users have never used Rails and they might not even know what Rack is. They think "here's my local machine and here are the override settings for my computer" and "there's github pages and here are the override settings for them". The idea of `development`, `test`, `production`, etc are all foreign. I think `local` is a more approachable term (and it has fewer characters!) for newcomers. I agree that it's a divergence from Rails and Rack, but my gamble is that the semantic benefit of `local` will help. So a site might have:

```text
.
| 
| - _config.yml
| - _config.local.yml
| - _config.github.yml
```

And that'd be it.